lib/modules/manager/pep621/processors/uv.ts

```diff
@@ -191,3 +199,34 @@ function generateCMD(updatedDeps: Upgrade[]): string {
 
   return `${uvUpdateCMD} ${deps.map((dep) => `--upgrade-package ${quote(dep)}`).join(' ')}`;
 }
+
+function getMatchingHostRule(url: string | undefined): HostRule {
+  const scopedMatch = find({ hostType: PypiDatasource.id, url });
+  return is.nonEmptyObject(scopedMatch) ? scopedMatch : find({ url });
```

Comment:
are you sure to need this fallback? I thought the find is already doing that.

@rarkins do you know more?

Reply:
I copy-pasted this from the poetry processor 😇 (which I guess doesn't prove anything... 😊)

Reply:
I think it's unnecessary because find() already does a fallback